src/main/java/com/cst438/service/GradebookServiceProxy.java

```diff
@@ -0,0 +1,100 @@
+package com.cst438.service;
+
+import com.cst438.domain.Enrollment;
+import com.cst438.domain.EnrollmentRepository;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Service;
+import com.cst438.dto.CourseDTO;
+import com.cst438.dto.SectionDTO;
+import com.cst438.dto.UserDTO;
+import com.cst438.dto.EnrollmentDTO;
+
+@Service
+public class GradebookServiceProxy {
+
+    Queue gradebookServiceQueue = new Queue("gradebook_service", true);
+
+    @Bean
+    public Queue createQueue() {
+        return new Queue("registrar_service", true);
+    }
+
+    @Autowired
+    RabbitTemplate rabbitTemplate;
+    EnrollmentRepository enrollmentRepository;
```

Comment:
Need to annotate Autowired before EnrollmentRepository.

Reply:
Updated!